### PR TITLE
Update screenshots and instructions in Journalist Guide

### LINKED ISSUE #4120: [docs] Most Tails screenshots in Journalist Guide are outdated
The Tails screenshots in the [journalist guide](https://docs.securedrop.org/en/latest/journalist.html) no longer reflect the current look and feel of Tails; Nautilus in particular looks very different now, and all window decorations have changed.

The screenshots need to be updated, and the guide needs to be vetted for accuracy against the current version of Tails.

### LINKED ISSUE #4560: Journalist Guide explanation re: Tor browser behavior is incorrect
The Journalist Guide states:

> In order to protect you from malware, the browser in Tails will only allow you to download documents to a special persistent folder located at `/home/amnesia/Tor Browser.`

> The special folder mentioned here is called Tor Browser, not “Persistent.” Attempting to download directly into the Persistent folder will only lead to frustration.

This is incorrect (it's unclear if it ever was correct, but the also very old screenshot suggests that it wasn't). The persistent folder is in `/home/amnesia/Persistent/Tor Browser`, while the non-persistent folder is `/home/amnesia/Tor Browser`.

### LINKED ISSUE #4565: Guidance re: GPG usage for journalists is confusing
The Journalist Guide instructs sources to generate individual GPG keys:

https://docs.securedrop.org/en/release-0.13.1/journalist.html#creating-a-gpg-key

This is framed as a must-have requirement ("Each journalist needs a personal GPG key for encrypting files"). The source is instructed to keep the private key on their *Journalist Workstation*, which we have defined in the glossary as "a machine that is online and used together with the Tails operating system on the online USB stick".

The guide later states:

> Plug the Transfer Device into your workstation computer and copy over the encrypted documents. Decrypt them with GPG. You are now ready to write articles and blog posts, edit video and audio, and begin publishing important, high-impact work! 

The use of "your workstation" here is ambiguous. Does this refer to their everyday workstation? If they've interpreted the previous instructions as "Keep the key on my Tails drive", that won't work. But the number of journalists who write articles and blog posts, edit video and audio, and publish important high-impact work while using Tails can likely be counted on one hand.

The current state is doubly problematic: it's highly prescriptive (you need to do this) while also being ambiguous and inconsistent. Few things put a reader at unease more than ambiguous yet highly prescriptive language ("You must wear the right clothing, or you'll suffocate under these conditions").

This is related to #2382, but some small improvements would go a long way: make the language less prescriptive (we're really just describing one possible workflow) and less ambiguous -- we *are* talking about the journalist's everyday workstation, no other option makes sense for the vast majority of users. 

This is also related to #4421 and #4449, which describe our confusing usage of "journalist key" in parts of the UI and codebase. Here we are talking about an actual "journalist key", but we need to be extra careful not to cause confusion with the other (hopefully soon to be deprecated) references to "journalist keys".

### PR DESCRIPTION
**Pull request by eloquence**
## Status

Ready for review

## Description of Changes

Resolves #4120
Resolves #4565
Resolves #4560

- Updated all Tails screenshots
- Clarified Tor Browser behavior for folder access
- Explained file name extraction issue with gzip files
- More pragmatic and consistent recommendation for GPG usage
- Added a section explaining folder hierarchy
- Added a section explaining risks of photographing documents
- Various smaller fixes

## Checklint
- [X] `make docs-lint` doesn't cry

### CODE REVIEW

**Comment by eloquence** [comment]
Making a note that this PR would benefit from review by @huertanix specifically for the Tor browser download instructions.

File: docs/journalist.rst [hunk 1]
```diff
@@ -180,31 +180,66 @@ and save them all at once by clicking **Download Selected**.
...
+
+- **Tor Browser**. Downloads saved to this folder will be stored in memory,
+  which means that they will only be available for the duration of your current
+  Tails session. In the screenshot, this is the currently selected folder.
+  The full path to this folder is ``/home/amnesia/Tor Browser``.
+
+- **Tor Browser (persistent)**: Note that the name may be shown in truncated
+  form, as seen in the screenshot. Downloads saved to this folder will be stored
+  on your Tails USB drive in the special persistent volume that is only
+  available if you have unlocked it on the welcome screen. The full path to this
```
Comment (eloquence) [inline docs/journalist.rst]: Note: change this to "Tails welcome screen" or similar

**Comment by huertanix** [comment]
I can't preview the full document in this branch due to some dev env issues, so I don't have a fully-informed opinion on what verbiage makes sense in the context of the whole page, but I would maybe start the note with mentioning that the `Tor Browser (persistent)` folder will only be available when persistence has been switched on at the Tails welcome screen, and then maybe after then mention how things saved there will be saved for future sessions (otherwise readers might be confused as to why they don't see the persistent folder[s]). Also wondering if _truncated_ might be too jargony, would aim for explaining it as part of the UI, e.g. something like "The left column (or whatever the area is called in Nautilus) might be too short to display the full folder name but you can hover over any folder name with your cursor to see the full thing."

In general, journalists given a "this may be a thing to look out for" note will immediately think or verbally ask "ok, so what do I do then?" so adding the cursor hover instruction or some other preferred way to view the full folder name would be helpful to include.

**Comment by huertanix** [comment]
Just viewed the full doc. Having the `(...` part of the file name in the screenshot is helpful; I would still suggest mentioning that the full name can be viewed by hovering over it, although maybe in a separate protip-style note near it if including that in the folder description might make it too wordy.

**Comment by eloquence** [comment]
Thanks @huertanix for taking a spin :)

> I would maybe start the note with mentioning that the Tor Browser (persistent) folder will only be available when persistence has been switched on at the Tails welcome screen

I didn't want to over-emphasize this part because my sense is that the user wouldn't even be able to get to this point without having unlocked their persistent volume (they won't have their Journalist Interface .onion address, its required HidServAuth token, or their login credentials) .

> wondering if truncated might be too jargony,

On second thought it definitely is. :) Will fix.

> I would still suggest mentioning that the full name can be viewed by hovering over it

Makes sense! Will tweak accordingly.

File: docs/journalist.rst [hunk 2]
```diff
@@ -248,33 +336,76 @@ default application.
...
-also need to right-click on a file and choose **Rename...** to rename
-a document with a proper file extension (for example, ".jpg" instead
-of ".jpeg").
-
 Working with Documents
 ----------------------
 
-This section describes how to handle unusual file formats, safely research
-submissions, remove metadata, and mitigate risks from submitted malware.
+This section describes how to organize submissions, handle unusual file formats,
```
Comment (emkll) [inline docs/journalist.rst]: Although implied by this paragraph, I recommend  be slightly more explicit and state that opening random attachment will always carry  certain risks

File: docs/journalist.rst [hunk 3]
```diff
@@ -375,16 +518,17 @@ such as an editor's) and click **OK**.
 |Encrypting 2|
 
 When you are done encrypting, you will have another document with the
-same filename but ending in ".gpg". This file is encrypted to the GPG
+same filename but ending in ``.pgp`` (not ``.gpg``; the ``.pgp`` extension is
+just another way to refer to the same format). This file is encrypted to the GPG
 keys you selected. You can safely copy these encrypted files to the
-*Transfer Device* to transfer them to your workstation.
+*Transfer Device* to transfer them to your everyday workstation.
```
Comment (emkll) [inline docs/journalist.rst]: Definitely worth reiterating the risks of opening random attachments here

File: docs/journalist.rst [hunk 4]
```diff
@@ -172,39 +182,82 @@ generated and you can log back in and send a reply.
...
-Documents sent by sources can only be viewed on the *Secure Viewing
-Station*. After clicking on an individual source, you will see the
-page below with any messages that source has sent you. Click on a
-document or message name to save it, or select a number of documents
+Step 1: Download the encrypted submission
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Documents and messages sent by sources can only be decrypted and viewed on the
+*Secure Viewing Station*. After clicking on an individual source, you will see the
+page below with any documents or messages that source has sent you. Documents
```
[the remark this answers is not in the document]
Comment (eloquence) [reply docs/journalist.rst]: intended meaning was *that source* ("that particular source"), changing to *the* for clarity

File: docs/journalist.rst [hunk 5]
```diff
@@ -172,39 +182,82 @@ generated and you can log back in and send a reply.
...
+of the two folders highlighted in red in the screenshot below:
 
 |Download to sandbox folder|
 
-.. tip:: The special folder mentioned here is called **Tor Browser**,
-         not "Persistent." Attempting to download directly into the
-         **Persistent** folder will only lead to frustration.
+The difference between these two folders is as follows:
+
+- **Tor Browser**. Downloads saved to this folder will be stored in memory,
```
[the remark this answers is not in the document]
Comment (eloquence) [reply docs/journalist.rst]: As far as I can tell, Tails uses a union-mounted [tmpfs](https://en.wikipedia.org/wiki/Tmpfs) filesystem.

```
tmpfs on /lib/live/mount/overlay type tmpfs (rw,noatime,mode=755)
```

Since it uses no swap, this should be equivalent to a RAM disk. That's consistent with the Tails docs:

> The only storage space used by Tails is in RAM, which is automatically erased when the computer shuts down.

https://tails.boum.org/about/index.en.html

Does that make sense? If that's correct, then I think being explicit about it being in-memory storage, it would be my preference to say so explicitly so users don't worry about wiping files that are stored in the non-persistent portion of the filesystem.

File: docs/journalist.rst [hunk 6]
```diff
@@ -172,39 +182,82 @@ generated and you can log back in and send a reply.
...
+  The full path to this folder is ``/home/amnesia/Tor Browser``.
+
+- **Tor Browser (persistent)**: Note that the name may be abbreviated, as shown
+  in the screenshot; you can view the full name by hovering the mouse over the
+  shortcut. Downloads saved to this folder will be stored
+  on your Tails USB drive in the special persistent volume that is only
+  available if you have unlocked it on the Tails welcome screen. The full path
+  to this folder is ``/home/amnesia/Persistent/Tor Browser``.
+
+Unless you have reason to store encrypted submissions on the
```
[the remark this answers is not in the document]
Comment (eloquence) [reply docs/journalist.rst]: Sounds good, tweaked accordingly.

File: docs/journalist.rst [hunk 7]
```diff
@@ -172,39 +182,82 @@ generated and you can log back in and send a reply.
...
+*Journalist Workstation*, we recommend using the non-persistent "Tor Browser"
+folder. In the recommended process, you will now move the submission to the
+*Secure Viewing Station*, and there is no need to leave a persistent copy
+behind.
+
+.. important:: Attempting to download files to any other folder will fail.
+  Tails only permits the browser to access these two folders, so that
```
[the remark this answers is not in the document]
Comment (eloquence) [reply docs/journalist.rst]: Done.

File: docs/journalist.rst [hunk 8]
```diff
@@ -172,39 +182,82 @@ generated and you can log back in and send a reply.
...
+Once downloaded to either folder, move the document to the designated
 USB stick you intend to use to transfer the documents from your
 *Journalist Workstation* to the *Secure Viewing Station*. This storage
-device will be known as your *Transfer Device*.
+device is known as your *Transfer Device*.
+
+You can drag and drop the file, as shown in the screenshot sequence below.
+This will leave a redundant copy behind in the Tor Browser folder. If you have
+downloaded the file to the non-persistent "Tor Browser" folder (as recommended),
+the redundant copy will disappear at the end of your session.
```
[the remark this answers is not in the document]
Comment (eloquence) [reply docs/journalist.rst]: changed to "when the computer is shut down or rebooted."

File: docs/journalist.rst (hunk 2 above)
Comment (eloquence) [reply docs/journalist.rst]: I've now expanded on the malware problem a bit in the malware section. The paragraphs starting with 
>"It is crucial, however,"
are new.

File: docs/journalist.rst [hunk 9]
```diff
@@ -172,39 +182,82 @@ generated and you can log back in and send a reply.
...
 
 |Move to transfer device 1|
 
 |Move to transfer device 2|
 
-Eject the *Transfer Device* from the *Journalist Workstation*.
+|Move to transfer device 3|
```
[the remark this answers is not in the document]
Comment (eloquence) [reply docs/journalist.rst]: I'm a bit wary of using GIFs here as the documentation may be printed out, which would make them useless. I've changed the screenshots and description to the "Copy to" flow you describe, I agree that's a bit easier to follow!

File: docs/journalist.rst (hunk 3 above)
Comment (eloquence) [reply docs/journalist.rst]: Added an "Important" warning re: malware, reiterating earlier points.

File: docs/journalist.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (emkll) [reply docs/journalist.rst]: That is correct, makes sense, thanks for the clarification
